src/components/main/Main.tsx

```diff
@@ -82,6 +84,7 @@ const Main = ({ children }: Props): JSX.Element => {
   );
   return (
     <GraaspMain
+      open={currentMember ? undefined : false}
```

Comment:
```suggestion
      open={Boolean(currentMember)}
```

Reply:
I want to use `undefined` so the default behaviour can be kept, i.e. when on mobile it will be closed, otherwise mobile users that are logged in will always have it open.